gemfiles/rails_7.0.gemfile

```diff
@@ -0,0 +1,8 @@
+# This file was generated by Appraisal
+
+source "http://rubygems.org"
+
+gem "sqlite3", "~> 1.4"
+gem "rails", "~> 7.0"
```

Comment:
is the full `rails` required, in the past i tried to decrease dependency just to activerecord